library/src/main/java/de/agilecoders/wicket/webjars/util/RecentVersionCallable.java

```diff
@@ -47,10 +45,8 @@ public String call() throws Exception {
      * @return recent version string
      */
     private static String collectRecentVersionFor(final String partialPath) {
-        return Holder.recentVersionProvider.findRecentVersionFor(partialPath);
+        AssetsMap assetsMap = AssetsMap.get();
+        return assetsMap.findRecentVersionFor(partialPath);
     }
 
-    static final class Holder {
```

Comment:
wicket-webjars loaded a long with wicket in container's class loader. But all plugins are loaded in child classloader which used during restart. This child classloader,as usual, transfer loading of known classes to parent: so wicket-webjars classes loaded once in parent classloader, but due to different child classloaders we need to have ability to refresh wicket-webjars cache during every app restart.